include/class.ostsession.php

```diff
@@ -40,7 +40,7 @@ function __construct($ttl=0){
         ini_set('session.gc_maxlifetime', $ttl);
 
         // Skip db version check if version is later than 1.7
-        if (!defined('MAJOR_VERSION') && OsticketConfig::getDBVersion())
+        if (OsticketConfig::getDBVersion())
```

Comment:
Why this change? Removing this will results in every page load checking DB version - which results in an SQL errorand unnecessary query for installation after `v1.7`.